portfolio/src/main/webapp/contact.html

```diff
@@ -36,6 +36,13 @@ <h1>Comments</h1>
 
         <!-- button to show comments-->
         <br><br>
+        <label for="limit">Number of comments to show:</label>
+        <select name="limit" id="limit">
+          <option value="1">1</option>
```

Comment:
maybe add another value (say -1) to view all of them? There's no way to do so and they are always limit to the maximum of 10 comments